operator/config/istio/virtual_service.yaml

```
@@ -0,0 +1,29 @@
+
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: kyma-events
+  namespace: operator-system
+spec:
+  gateways:
+    - default/kyma-gateway
+  hosts:
+    - '*'
+  http:
+    - match:
+        - uri:
+            regex: \/v.*\/kyma\/event
+      route:
+        - destination:
+            host: operator-kyma-event-service
```

```suggestion
            host: lifecycle-manager-kyma-event-service
```
This `operator-` prefix already changed, I think we also forgot to change it